src/templates/kit/screens/FundraisingScreen.js

```diff
@@ -270,19 +177,53 @@ function FundraisingScreen({
     return true
   }, [fields, minimumGrowth])
 
+  const fieldsAreValid = useMemo(() => {
+    const error = validationError(fields)
+    setFormError(error)
```

Comment:
Haha, I never thought that you could do this. This works, but in a more React-y fashion, you would split this side effect into a `useEffect()`.

Reply:
Ahahah. Actually I think this part is mostly inherited from Pierre's first iteration :)

Reply:
Ahh actually now I get what we were going for with this. It's to only show the form validation error on submits, not as the user's actually entering values.